lib/modules/commentPreview.js

```diff
@@ -119,13 +119,11 @@ module.beforeLoad = () => {
 module.go = () => {
 	if (module.options.enableBigEditor.value) {
 		// Install the 2 column editor
-		addBigEditorListener();
+		$('body').on('click', '.RESBigEditorPop', showBigEditor);
 
 		if (Modules.isRunning(KeyboardNav)) {
-			$('body').on('keydown', '.usertext-edit textarea, #wiki_page_content', (e: KeyboardEvent) => {
-				if (checkKeysForEvent(e, module.options.openBigEditor.value)) {
-					showBigEditor(e);
-				}
+			$('body').on('keydown', '.usertext-edit textarea, #wiki_page_content', function(e: KeyboardEvent) {
+				if (checkKeysForEvent(e, module.options.openBigEditor.value)) showBigEditor.bind(this)(e);
```

Comment:
Can we change it to use `e.target`? Looks like it should be the same as `this` in both listeners (if it isn't, we find the textarea with `.parents(...).find()` so it should still work).